ui/src/components/VaultSummary.jsx

```diff
@@ -94,14 +105,22 @@ export function VaultSummary({ vault, brandToInfo, id }) {
   } = makeDisplayFunctions(brandToInfo);
 
   const {
-    debtSnapshot, // amount
```

Comment:
+1 to removing the type comments. they could get out of sync with the real type and it's easy enough to see the type on hover